treeple/stats/utils.py

```diff
@@ -316,3 +329,209 @@ def get_per_tree_oob_samples(est: BaseForest):
         )
         oob_samples.append(unsampled_indices)
     return oob_samples
+
+
+def _get_forest_preds_sparse(
```

Comment:
maybe I missed it somewhere, but are there unit tests for these methods anywhere?

Reply:
@ryanhausen

Reply:
@sampan501, @adam2392  No there isn't a test for this function in test_utils.py. This function, `_get_forest_preds_sparse`, is a good candidate for a unit test though. I can write one up. 

I worry the other functions: `_parallel_build_null_forests_sparse` and `_compute_null_distribution_coleman_sparse`, like their dense counterparts, do too much to be easily unit tested and are tested via the integration tests. However,  if you would like, we can refactor the dense/sparse functions into smaller units so that they can all have tests written. Would probably be cleaner, but would take time. 

What would you like to see?

Reply:
For now, I think it's fine the way it is. If we find means of re-designing the internal API to make things simpler and easier to test, then it's warranted, but I don't see a clear path. Do you?

Reply:
Yeah I think that's sufficient then for now.

Reply:
sounds good. Maybe we can make an issue for having such unit tests/re-designing the API so we don't forget. Otherwise, I think this PR lgtm